### PR TITLE
[Layout] Update navigation bar to responsive

### PR DESCRIPTION
**Pull request by sureemo**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

Result (for screen width less than 630px):
navbar-responsive-improved

### CODE REVIEW

File: client/src/components/Header/header.css [hunk 1]
```diff
@@ -121,3 +115,25 @@ li.user-state-link > a:focus {
...
+  }  
+
+  .fcc_searchBar {
+    padding: 3px 53px 0 10px;
+    position: absolute;
+    top: 36px;
+    width: 100%;
+  }
+
+  .app-wrapper {
```
Comment (Bouncey) [inline client/src/components/Header/header.css]: I don't think this class is available any more. You would perhaps need to apply these styles in the `Default` layout as all layouts use this as a base.

**Review by Bouncey — APPROVED** [review]
Sorry for the delay @sureemo. The screen shot looks good, but things have changed since this PR has been raised

**Review by Bouncey — CHANGES_REQUESTED** [review]
I hit the wrong button before, sorry

**Comment by RandellDawson** [comment]
There is an outstanding issue #29824 regarding what is being attempted to resolve here.  

@Bouncey FYI - There is a newer PR #34586 which attempts a fix for this issue.

**Comment by RandellDawson** [comment]
@Bouncey Are these changes still relevant or should this PR be closed?

**Comment by raisedadead** [comment]
We have taken a different approach to this and the search also needs some more work.

@sureemo 

Thanks a lot for taking the time to prepare this PR. Unfortunately we haven't been able to land this in time. Please do not be disappointed. We are really grateful for your contribution and would love to see some more in future.

Happy contributing.

**Pull request closed without merging**